src/components/meeting/JitsiComponentWrapper.js

```diff
@@ -1,9 +1,38 @@
 import {JitsiComponent} from "e-stella-jitsi";
+import {MeetingFab} from "./MeetingFab";
+import {useState} from "react";
+import {Whiteboard} from "e-stella-whiteboard";
 
 export const JitsiWrapper = (props) => {
+
+    const [jitsiHeight, setJitsiHeight] = useState('calc(100vh - 60px)')
+    const [jitsiWidth, setJitsiWidth] = useState('100%')
+    const [whiteboardIsSeen, setWhiteboardIsSeen] = useState(false)
+
+    const addWhiteboard = () => {
+        setJitsiWidth('20%')
+        setJitsiHeight('calc(100vh - 60px)')
+        setWhiteboardIsSeen(true)
+    }
+
+    const defaultView = () => {
+        setJitsiHeight('calc(100vh-60px)')
+        setJitsiWidth('100%')
+        setWhiteboardIsSeen(false)
+    }
+
+    const getWhiteboardCode = () => {
+        if(props.roomName.length < 15) return 'abcdefghijklmnopqrst'
```

Comment:
Czy to czysto teoretycznie nie powinien być secret?

Reply:
roomName nie, ale client Key jak najbardziej XD